### ISSUE TITLE
Layout Transformer Quantitative Evaluation

### ISSUE THREAD

**Issue by abhijeet00** [Layout-Generation/layout-generation#6]
<img width="576" alt="Screenshot 2021-11-18 at 5 02 34 PM" src="https://user-images.githubusercontent.com/30382260/142408015-158566a6-d3a8-4d37-b842-ab7f71f654cf.png">

Overlap, IOU, and Alignment loss of the Layout transformer is close to 1 for all these metrics.
Also, there is a high difference in metrics values between other models.

As per layout_completion function in Layout_Transformer.ipynb , a training set(starting two elements of each file) was used for layout generation. There is high chances that model is memorising the training dataset.

Can you please get the metrics for validation set to check if the metrics values are same for layout transformer?

Thank you!

**Comment by 23yashm** [comment]
The hardware on which we ran the code is **not** available with us at the moment, so we can't calculate the metrics. As soon we get the hardware back, we will calculate it.

But earlier, we generated some of the documents (using one of the element from documents **outside the training set**). Below are the results.

![output_5 (on unseen data)](https://user-images.githubusercontent.com/67385364/143292998-97e79601-cb86-43a3-9eb8-03bc49012b31.png)

The predicted documents have a great diversity and are not similar to the inputs, so we can say that the model is _not overfitting_.

_Thank You!_
